### PR TITLE
Upgraded react-native to v0.27.2

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

File: lib/views/list-item/index.js [hunk 1]
```diff
@@ -141,13 +141,13 @@ class ListItem extends React.Component {
...
-        debug('on responser grant', nativeEvent);
+      onResponderGrant() {
+        self.tapStart = Date.now();
+        debug('on responser grant');
       },
 
-      onResponderRelease({nativeEvent}) {
-        var time = getTime(nativeEvent) - self.tapStart;
+      onResponderRelease() {
+        var time = Date.now() - self.tapStart;
```
Comment (wilsonpage) [inline lib/views/list-item/index.js:150]: For some reasons the timestamps we were getting from the `nativeEvent` objects we're incorrect. `Date.now()` seemed to work more reliably.

**Comment by wilsonpage** [comment]
@samgiles r?

File: package.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
     "eslint": "^2.5.3",
     "events": "^1.1.0",
     "magnet-html-parser": "github:mozilla-magnet/magnet-html-parser#v2.0.1",
-    "react": "^0.14.0",
-    "react-native": "^0.25.0",
+    "react": "^15.1.0",
```
Comment (samgiles) [inline package.json:17]: Did versioning change, or were we that far behind?!

Comment (wilsonpage) [inline package.json:17]: Yeah they dropped the `0.`

**Pull request merged**